backend/tournesol/views/inconsistencies.py

```diff
@@ -302,9 +302,11 @@ def _list_inconsistent_comparisons(criteria_comparisons: list,
 
         inconsistent_criteria_comparisons.sort(key=lambda d: d['inconsistency'], reverse=True)
 
-        response["count"] = len(inconsistent_criteria_comparisons)
-        response["results"] = inconsistent_criteria_comparisons
-        response["stats"] = {}
+        response = {
+            "count": len(inconsistent_criteria_comparisons),
+            "results": inconsistent_criteria_comparisons,
+            "stats": {},
+        }
```

Comment:
This fixes no warnings, I just though it was more readable.
I deactivated the warning "too-many-locals" (21/15 local variables) in this function, not sure it would be more readable after a refactoring.